Makefile

```diff
@@ -18,6 +18,13 @@ export DOCKER=docker.exe
 export RUN_DOCKER_ENGINE_ROOT=1
 export DOCKER_GID=1042
 export HOST_GID=1000
+else ifneq (,$(findstring Darwin,$(VERSION)))
+$(info    detected OSX)
+export DOCKER_COMPOSE=docker-compose
+export DOCKER=docker
+export RUN_DOCKER_ENGINE_ROOT=1
+export DOCKER_GID=1042
```

Comment:
these variables are already defined in .venv-devel, oder?

Reply:
DOCKER_GID is defined, but I guess a linux user might want to override them with whatever is defined on his/her machine?
RUN_DOCKER_ENGINE_ROOT is defined as 0, but needs to be 1 for any non-linux host.

Reply:
this PR just adds support for osx ... lets fix the deeper inconsistencies in a separate one